packages/preview-middleware/src/base/flp.ts

```diff
@@ -227,14 +229,17 @@ export class FlpSandbox {
                 previewUrl = `${previewUrl}.inner.html`;
                 editor.pluginScript ??= 'open/ux/preview/client/cpe/init';
                 this.router.get(editor.path, (_req: Request, res: Response) => {
+                    const scenario = rta.options?.scenario;
                     let templatePreviewUrl = `${previewUrl}?sap-ui-xx-viewCache=false&fiori-tools-rta-mode=forAdaptation&sap-ui-rta-skip-flex-validation=true&sap-ui-xx-condense-changes=true#${this.config.intent.object}-${this.config.intent.action}`;
-                    if (rta.options?.scenario === 'ADAPTATION_PROJECT') {
+                    if (scenario === 'ADAPTATION_PROJECT') {
                         templatePreviewUrl = templatePreviewUrl.replace('?', `?sap-ui-layer=${rta.layer}&`);
                     }
                     const template = readFileSync(join(__dirname, '../../templates/flp/editor.html'), 'utf-8');
                     const html = render(template, {
                         previewUrl: templatePreviewUrl,
-                        telemetry: rta.options?.telemetry ?? false
+                        telemetry: rta.options?.telemetry ?? false,
+                        appName: rta.options?.appName ?? '',
+                        scenario
```

Comment:
Just a suggestion. Would it be better to have default appName different than empty string?

Reply:
The manifest will always be present, and from it we will retrieve the id (componentId) and the title, like [here](https://github.com/SAP/open-ux-tools/pull/1553/files/7d940d05be3b9273c24f399c199d778e10e20620#diff-b39fc9ea1aedc0aa1072f86f46fe70f7cf01422c26f13952e9fd1dcffca171a3R179). So I am expecting there will always be an id (title also but not really sure). If there is no id or title than it will default to empty string. In that case it will be `Editing` as the title in the browser tab, which is fine. And I don't think there will be a case like this.